documentation/userguide.md

```diff
@@ -95,12 +95,39 @@ After restarting the IIS/recycling the pool and opening the application in a bro
 
 ### Azure Cloud
 
-Azure instantly kills the application when it is stopped/restarted, thus the profiler has no time to write anything to the trace file. To remedy this, enable eager mode (see the Configuration section. Please note that eager mode should only be used together with light mode as it will otherwise drastically increase lag during the application startup.
-
-To deploy the profiler to the Azure cloud, enable the FTP account. Store the profiler in a directory that will not be overwritten by a new deployment, e.g. `D:\home\site\repository\profiler\`
-Also create a new directory to write the traces to, e.g. `D:\home\site\repository\traces\`
-
-To set the environment variables in Azure, go to the application settings and add them as new application settings entries.
+If the application is running in an Azure App Service, the following steps need to be taken:
+
+**Install the Profiler:**
```

Comment:
Please make this a level 4 heading, not a bold line. Also please don't use use colons at the end of headings. Same for the other subheadings below